python/lsst/verify/extract_metricvalues.py

```diff
@@ -96,7 +97,8 @@ def print_diff_metrics(butler1, butler2, data_id_keys=None, verbose=False):
         Butlers to load values to difference from.
     data_id_keys : `collection` [`str`], optional
         List of Butler dataId keys to restrict the printed output to;
-        for example: ``("detector", "visit")``.
+        for example: ``("detector", "visit")``. If a metric does not include a
+        key listed here, it is printed with default formatting.
```

Comment:
"does not use all of the keys"?

(Am I understanding correctly that, in the example, a metric that uses visit but not detector gets default-printed?)

Reply:
Yes, your understanding is correct. I decided it was better to default print than not print at all, if there's a print format restriction that doesn't apply.